pkg/porter/show.go

```diff
@@ -163,27 +171,23 @@ func (p *Porter) ShowInstallation(ctx context.Context, opts ShowOptions) error {
 	}
 }
 
-func (p *Porter) generateDisplayInstallation(ctx context.Context, installation storage.Installation) (DisplayInstallation, error) {
-	run, err := p.Installations.GetRun(ctx, installation.Status.RunID)
-	if err != nil {
-		return DisplayInstallation{}, err
-	}
-
+func (p *Porter) NewDisplayInstallationWithSecrets(ctx context.Context, installation storage.Installation, run *storage.Run) (DisplayInstallation, error) {
 	bun := cnab.ExtendedBundle{run.Bundle}
```

Comment:
What if the run is nil here?

Reply:
Haha you caught that at the same time I did, I was fixing that right before our meeting this morning. I just pushed it up now.